frontend/pages/index.js

```diff
@@ -123,21 +126,40 @@ export default function Index({ projects, organizations }) {
 
 Index.getInitialProps = async ctx => {
   const { token } = Cookies(ctx);
+  if (!token) {
+    console.log(`Error: Token was ${token}...`)
+  }
+
   return {
     projects: await getProjects(token),
     organizations: await getOrganizations(token)
   };
 };
 
 const getProjects = async token => {
+  // console.log("In getProjects...")
   try {
+    // Read local API URL. This should hit the Django endpoint?
+    // That's the featured projects endpoint?
+
+    const featuredProjectsEndpoint = `${process.env.API_URL}/api/featured_projects/`;
+
+    console.log(`Hitting featured projects endpoint: ${featuredProjectsEndpoint}`);
+
+
     const resp = await axios.get(
-      process.env.API_URL + "/api/featured_projects/",
+      featuredProjectsEndpoint,
       tokenConfig(token)
     );
-    if (resp.data.length === 0) return null;
-    else return parseProjects(resp.data.results);
+
+    if (resp.data.length === 0) {
+      // console.log("Null 🗻")
+      return null;
+    }
+
+    return parseProjects(resp.data.results);
   } catch (err) {
+    console.log("🔴");
```

Comment:
Do we need this console.log here?

Reply:
Definitely not :D thanks for catching this!